components/file/s3/aws/oss_test.go

```diff
@@ -96,6 +96,25 @@ func TestAwsOss_Put(t *testing.T) {
 
 	req.FileName = "/a.txt"
 	err = oss.Put(context.Background(), req)
-	assert.Equal(t, err.Error(), "awsoss put file[/a.txt] fail,err: invalid fileName format")
```

Comment:
Why delete this assertion?

Reply:
Now this assertion can not pass without environment, can be added later

Reply:
OK. The existing test cases depend on aws environment. We can refactor them with mocks in the future

Reply:
@akkw Hi, I read through the code and found that we don't have to delete this line.
The existing ut don't have dependency on remote aws server. This error is raised by https://github.com/mosn/layotto/blob/223945e4a49c499ac66a5c13b267af6b080a79a1/components/file/s3/aws/oss.go#L160

So, the s3 component get this error before connecting to the remote aws server, and we don't need an aws s3 account to run the UT

Reply:
Yes, it is now connected to the remote service